### PR TITLE
Update Drag&Drop code

### PR DESCRIPTION
**Pull request by nmorey**
- Allow moving containers (Issue #7)
- Allow moving tab in another container in a given position

Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>

### CODE REVIEW

File: src/sidebar/style.css [hunk 1]
```diff
@@ -170,16 +170,19 @@ li {
...
+.container-dragged-over .container-header,
+.container-dragged-over .container-tab,
+.tab-dragged-over.container-tab,
+.tab-dragged-over .container-header
+ {
     border-bottom: 1px solid #eee !important;
 }
 
-#pinned-tabs .container-tab-dragged-over {
-    border-right: 1px solid #eee;
```
Comment (maciekmm) [inline src/sidebar/style.css]: This border-right was used when moving tab from unpinned to a pinned state.  Now there's no indicator where the tab will land.

File: src/sidebar/containers/container.js [hunk 2]
```diff
@@ -48,12 +48,20 @@ class AbstractTabContainer {
         this.element.addEventListener('dragleave', (e) => {
             if (!e.currentTarget || !e.currentTarget.classList) return
             e.currentTarget.classList.remove('container-dragged-over')
-        })
+            e.currentTarget.classList.remove('tab-dragged-over')
```
Comment (maciekmm) [inline src/sidebar/containers/container.js]: Could you change the indention to spaces? The codebase uses space indention, and while I prefer tabs I would rather not mix the two.

File: src/sidebar/style.css (hunk 1 above)
Comment (nmorey) [reply src/sidebar/style.css]: OK

File: src/sidebar/containers/container.js (hunk 2 above)
Comment (nmorey) [reply src/sidebar/containers/container.js]: Sure.

**Comment by nmorey** [comment]
> We might think about saving the container ordering in some sort of storage.

It would make a lot of sense. Just switching to bookmarks in the sidebar and going back to the tabs resets the order.
But I'm bery new to JS development so I'm spending way too much time trying to figure out what API to use.

**Comment by maciekmm** [comment]
Hello, 
I implemented position setting when dragging between containers. Will close this pull request for now. Thank You for your contribution.

**Pull request closed without merging**